astropy/coordinates/frame_attributes.py

```diff
@@ -363,13 +363,21 @@ def convert_input(self, value):
 
             if not hasattr(value, 'transform_to'):
                 raise ValueError('"{0}" was passed into an '
-                                 'EarthLocationAttribute, but it does not have '
+                                 'EarthLocationFrameAttribute, but it does not have '
                                  '"transform_to" method'.format(value))
             itrsobj = value.transform_to(ITRS)
             return itrsobj.earth_location, True
 
 
-class CoordinateAttribute(FrameAttribute):
+class EarthLocationAttribute(EarthLocationFrameAttribute):
+
+    def __init__(self, **kwargs):
+        warnings.warn("EarthLocationAttribute has been renamed to EarthLocationFrameAttribute.",
+        AstropyDeprecationWarning)
```

Comment:
When you address this, add `[ci skip]` to the commit message so it skips the CI checks.